lib/pipedrive/api_operations/request.rb

```diff
@@ -27,7 +27,11 @@ def api_client
           @api_client = Faraday.new(
             url: BASE_URL,
             headers: { 'Content-Type': "application/json" }
-          )
+          ) do |faraday|
+            if Pipedrive.debug_api
+              faraday.response :logger
+            end
```

Comment:
```suggestion
            faraday.response :logger if Pipedrive.debug_api
```
This is what rubocop suggest 🙂